app/models/team.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 class Team < ApplicationRecord
-  has_many :home_matches, class_name: 'Match', foreign_key: 'team_home_id'
-  has_many :away_matches, class_name: 'Match', foreign_key: 'team_away_id'
+  has_many :home_matches, class_name: 'Match', foreign_key: 'team_home_id', dependent: :restrict_with_exception # rubocop:disable Rails/InverseOf
```

Comment:
@csalvato, I couldn't find a way to pass the Rubocop. The inverse of does exist but somehow the linter doesn't acknowledge it.

Reply:
Because the association does not match a class directly, and there's two associations on the other class (`Match`), it doesn't know what the inverse of this relationship is, and it can't infer it.

What you want is:

```rb
has_many :home_matches, class_name: 'Match', foreign_key: 'team_home_id', dependent: :restrict_with_exception, inverse_of: :team_home
  has_many :away_matches, class_name: 'Match', foreign_key: 'team_away_id', dependent: :restrict_with_exception, inverse_of: :team_away
```

This tells Rails that the inverse of this association is the home team and away team, respectively. Without that hint, Rails doesn't know which team association belongs to which match assocation.

Reply:
Wow. Hashed arguments. Also argument rich but hard discoverability. Thanks!

Reply:
Resolved on #17